config.toml

```diff
@@ -45,17 +45,24 @@ time_format_blog = "Monday, January 02, 2006"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
-latest = "v1.10"
+latest = "v1.11"
 
-fullversion = "v1.10.3"
-version = "v1.10"
+fullversion = "v1.11.0"
+version = "v1.11"
 githubbranch = "master"
-docsbranch = "master"
+docsbranch = "release-1.11"
```

Comment:
`docsbranch` should remain `master` so that it is correct when the `release-1.11` branch gets merged into `master`.